templates/cf.yml

```diff
@@ -631,6 +654,9 @@ properties:
     blacklisted_syslog_ranges: ~
     unmarshaller_count: (( merge || 5 ))
     port: (( merge || 4443 ))
+    etcd:
+      client_cert: ~
+      client_key: ~
```

Comment:
@wfernandes Makes sense, want to allow different client certs for different jobs.  That's all well and good, but for the spiff templates, which are sorta geared to simplicity, they should probably just copy the same client cert around to all clients, rather than ask the new operator to produce N different client certs and keys.  So in spiff, I still stand by my original line comment here.